testing/coreidle/test_main_c.cpp

```diff
@@ -399,6 +394,36 @@ TEST_P(coreidle_main_c_p, parse_err2) {
   EXPECT_EQ(coreidle_main(13, argv), 0);
 }
 
+/**
+ * @test       invalid_main*
+ * @brief      Test: coreidle_main
+ * @details    When called with parameters will nullbytes,<br>
+ *             coreidle_main runs to completion, it fails to read <br>
+ *             .gbs file and returns zero upon successful cleanup.<br>
+ */
+TEST_P(coreidle_main_c_p, invalid_main0) {
+  char zero[32];
+  char one[32];
+  char two[32];
+  strcpy(zero, "coreidle");
+  strcpy(one, "-G");
+  strcpy(two, "temp\0.gbs");
+
+  char *argv[] = { zero, one, two };
+  EXPECT_EQ(coreidle_main(3, argv), 0);
+
+  memset(two, 0, 32);
+  strcpy(two, "\0temp.gbs");
+  EXPECT_EQ(coreidle_main(3, argv), 0);
```

Comment:
two will be the empty string "".  Seems like we should get an error, since there will be no parameter to the -G argument?